libs/libmilter/Makefile

```diff
@@ -0,0 +1,78 @@
+#
+# Copyright (C) 2017 Val Kulkov <val.kulkov@gmail.com>
+#
+# This is free software, licensed under the MIT License
+#
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=sendmail
+PKG_VERSION:=8.15.2
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME).$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=http://artfiles.org/sendmail.org/pub/sendmail/ ftp://ftp.cs.berkeley.edu/ucb/sendmail/ http://www.netgull.com/sendmail/ http://sendmail.cybermirror.org/
+PKG_HASH:=24f94b5fd76705f15897a78932a5f2439a32b1a2fdc35769bb1a5f5d9b4db439
+PKG_MAINTAINER:=Val Kulkov <val.kulkov@gmail.com>
+
+PKG_LICENSE:=Sendmail
+PKG_LICENSE_FILES:=LICENSE
+
+PKG_INSTALL:=1
+
+include $(INCLUDE_DIR)/package.mk
+
+MAKE_PATH:=libmilter
+SM_SITECONFIG:=devtools/Site/site.config.m4
+export MILTER_SOVER=1.0.1
```

Comment:
I do not like this export, can you try to add this to the MAKE_FLAGS or something like that